src/pages/[platform]/ai/conversation/tools/index.mdx

```diff
@@ -29,24 +29,41 @@ export function getStaticProps(context) {
 }
 
 
+Tools allow LLMs to query information to respond with current and relevant information. They are invoked only if the LLM requests to use one based on the user's message and the tool's description.
 
-Tools allow LLMs to take action or query information so it can respond with up to date information. There are a few different ways to define LLM tools in the Amplify AI kit.
+There are a few different ways to define LLM tools in the Amplify AI kit.
```

Comment:
```suggestion
There are three ways to define LLM tools in the Amplify AI kit.
```

a few feels too vague for documentation.

Reply:
There are actually four 🙈 
The last one is client tools. Let's update this to four when we add the client tool docs.